### PR TITLE
ADM1 Documentation Update 

### PR DESCRIPTION
**Pull request by MarcusHolly**
## Summary/Motivation:
Updates ADM1 and Modified ADM1 documentation based on changes made in #1038 

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

File: docs/technical_reference/property_models/ADM1.rst [hunk 1]
```diff
@@ -139,13 +139,14 @@ Kinetic Parameters
    "First-order decay rate for X_pro, k_dec_X_pro", ":math:`k_{dec,X_{pro}}`", "k_dec_X_pro", 0.02, ":math:`\text{d}^{-1}`"
    "First-order decay rate for X_ac, k_dec_X_ac", ":math:`k_{dec,X_{ac}}`", "k_dec_X_ac", 0.02, ":math:`\text{d}^{-1}`"
    "First-order decay rate for X_h2, k_dec_X_h2", ":math:`k_{dec,X_{h2}}`", "k_dec_X_h2", 0.02, ":math:`\text{d}^{-1}`"
-   "Water dissociation constant, KW", ":math:`KW`", "KW", 2.08e-14, ":math:`(\text{kmol/}\text{m}^3)^2`"
+   "Water dissociation constant, pKW", ":math:`pKW`", "pKW", 14, ":math:`\text{dimensionless}`"
```
Comment (adam-a-a) [inline docs/technical_reference/property_models/ADM1.rst]:
```suggestion
   "Water dissociation constant, pKW", ":math:`pK_W`", "pKW", 14, ":math:`\text{dimensionless}`"
```

Comment (adam-a-a) [reply docs/technical_reference/property_models/ADM1.rst]: I was debating whether we should explicitly describe in terms of "negative logarithm of ..." but I see below in constraints that that gets spelled out, so perhaps this is good enough.

File: docs/technical_reference/property_models/ADM1.rst [hunk 2]
```diff
@@ -185,28 +186,35 @@ Additional Variables
...
+   "Water dissociation constant constraint", ":math:`log(10^{-pKW}) = log(1e-14) + (\frac{55900}{R} * (\frac{1}{T_{ref}} - \frac{1}{T}))`"
+   "CO2 acid-base equilibrium constraint", ":math:`log(10^{-pK_{a,co2}}) = log(10^{-6.35}) + (\frac{7646}{R} * (\frac{1}{T_{ref}} - \frac{1}{T}))`"
+   "Nitrogen acid-base equilibrium constraint", ":math:`log(10^{-pK_{a,IN}}) = log(10^{-9.25}) + (\frac{51965}{R} * (\frac{1}{T_{ref}} - \frac{1}{T}))`"
+   "pH of solution", ":math:`pH = -log(S_{H})`"
+   "Mass concentration of valerate, va-", ":math:`C_{va,ref} = C_{va} (1 + \frac{S_{H}}{K_a_va})`"
+   "Mass concentration of butyrate, bu-", ":math:`C_{bu,ref} = C_{bu} (1 + \frac{S_{H}}{K_a_bu})`"
+   "Mass concentration of propionate, pro-", ":math:`C_{pro,ref} = C_{pro} (1 + \frac{S_{H}}{K_a_pro})`"
+   "Mass concentration of acetate, ac-", ":math:`C_{ac,ref} = C_{ac} (1 + \frac{S_{H}}{K_a_ac})`"
+   "Molar concentration of bicarbonate, HCO3", ":math:`pK_a_co2 = log(M_{co2}) - log(M_{hco3}) + pH`"
+   "Molar concentration of ammonia, NH3", ":math:`pK_a_IN = log(M_{nh4}) - log(M_{nh3}) + pH`"
```
Comment (adam-a-a) [inline docs/technical_reference/property_models/ADM1.rst]: I think you need curly braces for any math expressions that have more than one underscore. This is what is currently showing in readthedocs:
image

File: docs/technical_reference/property_models/ADM1.rst [hunk 3]
```diff
@@ -185,28 +186,35 @@ Additional Variables
...
-   "CO2 acid-base equilibrium constraint", ":math:`K_{a,co2} = 10^{-6.35} exp{(\frac{7646}{R} * (\frac{1}{T_{ref}} - \frac{1}{T}))}`"
-   "Nitrogen acid-base equilibrium constraint", ":math:`K_{a,IN} = 10^{-9.25} exp{(\frac{51965}{R} * (\frac{1}{T_{ref}} - \frac{1}{T}))}`"
-   "Mass concentration of valerate, va-", ":math:`C_{va} = \frac{K_{a,va} * C_{va,ref}}{K_{a,va} + S_{H}}`"
-   "Mass concentration of butyrate, bu-", ":math:`C_{bu} = \frac{K_{a,bu} * C_{bu,ref}}{K_{a,bu} + S_{H}}`"
-   "Mass concentration of propionate, pro-", ":math:`C_{pro} = \frac{K_{a,pro} * C_{pro,ref}}{K_{a,pro} + S_{H}}`"
-   "Mass concentration of acetate, ac-", ":math:`C_{ac} = \frac{K_{a,ac} * C_{ac,ref}}{K_{a,ac} + S_{H}}`"
-   "Molar concentration of bicarbonate, HCO3", ":math:`M_{hco3} = \frac{K_{a,co2} * \frac{C_{S_{IC},ref}}{12}}{K_{a,co2} + S_{H}}`"
-   "Molar concentration of ammonia, NH3", ":math:`M_{nh3} = \frac{K_{a,IN} * \frac{C_{S_{IN},ref}}{14}}{K_{a,IN} + S_{H}}`"
+   "Water dissociation constant constraint", ":math:`log(10^{-pKW}) = log(1e-14) + (\frac{55900}{R} * (\frac{1}{T_{ref}} - \frac{1}{T}))`"
```
Comment (adam-a-a) [inline docs/technical_reference/property_models/ADM1.rst]:
```suggestion
   "Water dissociation constant constraint", ":math:`log(10^{-pKW}) = log(10^{-14}) + (\frac{55900}{R} * (\frac{1}{T_{ref}} - \frac{1}{T}))`"
```

File: docs/technical_reference/property_models/modified_ADM1.rst [hunk 4]
```diff
@@ -262,22 +274,21 @@ Additional Constraints
...
-   "CO2 acid-base equilibrium constraint", ":math:`K_{a,co2} = 10^{-6.35} exp{(\frac{7646}{R} * (\frac{1}{T_{ref}} - \frac{1}{T}))}`"
-   "Nitrogen acid-base equilibrium constraint", ":math:`K_{a,IN} = 10^{-9.25} exp{(\frac{51965}{R} * (\frac{1}{T_{ref}} - \frac{1}{T}))}`"
-   "Mass concentration of valerate, va-", ":math:`C_{va} = \frac{K_{a,va} * C_{va,ref}}{K_{a,va} + S_{H}}`"
-   "Mass concentration of butyrate, bu-", ":math:`C_{bu} = \frac{K_{a,bu} * C_{bu,ref}}{K_{a,bu} + S_{H}}`"
-   "Mass concentration of propionate, pro-", ":math:`C_{pro} = \frac{K_{a,pro} * C_{pro,ref}}{K_{a,pro} + S_{H}}`"
-   "Mass concentration of acetate, ac-", ":math:`C_{ac} = \frac{K_{a,ac} * C_{ac,ref}}{K_{a,ac} + S_{H}}`"
-   "Molar concentration of bicarbonate, HCO3", ":math:`M_{hco3} = \frac{K_{a,co2} * \frac{C_{S_{IC},ref}}{12}}{K_{a,co2} + S_{H}}`"
-   "Molar concentration of ammonia, NH3", ":math:`M_{nh3} = \frac{K_{a,IN} * \frac{C_{S_{IN},ref}}{14}}{K_{a,IN} + S_{H}}`"
+   "Water dissociation constant constraint", ":math:`log(10^{-pKW}) = log(1e-14) + (\frac{55900}{R} * (\frac{1}{T_{ref}} - \frac{1}{T}))`"
```
Comment (adam-a-a) [inline docs/technical_reference/property_models/modified_ADM1.rst]:
```suggestion
   "Water dissociation constant constraint", ":math:`log(10^{-pKW}) = log(10^{-14}) + (\frac{55900}{R} * (\frac{1}{T_{ref}} - \frac{1}{T}))`"
```

File: docs/technical_reference/property_models/modified_ADM1.rst [hunk 5]
```diff
@@ -262,22 +274,21 @@ Additional Constraints
...
+   "CO2 acid-base equilibrium constraint", ":math:`log(10^{-pK_{a,co2}}) = log(10^{-6.35}) + (\frac{7646}{R} * (\frac{1}{T_{ref}} - \frac{1}{T}))`"
+   "Nitrogen acid-base equilibrium constraint", ":math:`log(10^{-pK_{a,IN}}) = log(10^{-9.25}) + (\frac{51965}{R} * (\frac{1}{T_{ref}} - \frac{1}{T}))`"
+   "pH of solution", ":math:`pH = -log(S_{H})`"
+   "Mass concentration of valerate, va-", ":math:`C_{va,ref} = C_{va} (1 + \frac{S_{H}}{K_a_va})`"
+   "Mass concentration of butyrate, bu-", ":math:`C_{bu,ref} = C_{bu} (1 + \frac{S_{H}}{K_a_bu})`"
+   "Mass concentration of propionate, pro-", ":math:`C_{pro,ref} = C_{pro} (1 + \frac{S_{H}}{K_a_pro})`"
+   "Mass concentration of acetate, ac-", ":math:`C_{ac,ref} = C_{ac} (1 + \frac{S_{H}}{K_a_ac})`"
+   "Molar concentration of bicarbonate, HCO3", ":math:`pK_a_co2 = log(M_{co2}) - log(M_{hco3}) + pH`"
+   "Molar concentration of ammonia, NH3", ":math:`pK_a_IN = log(M_{nh4}) - log(M_{nh3}) + pH`"
```
Comment (adam-a-a) [inline docs/technical_reference/property_models/modified_ADM1.rst]: Check these again for the issue with more than one underscore in certain math expressions, requiring curly braces for properly displaying in readthedocs (see previous comment)

**Review by adam-a-a — CHANGES_REQUESTED** [review]
Looks good- just some subtle changes to incorporate

**Review by adam-a-a — APPROVED** [review]

**Review by agarciadiego — APPROVED** [review]
LGTM

**Pull request merged**